oak/server/oak_manager.cc

```diff
@@ -65,22 +72,26 @@ void OakManager::CreateEnclave(const std::string& node_id, const std::string& mo
   initialize_input->set_module(module);
   asylo::Status status = enclave_manager_->LoadEnclave(node_id, *enclave_loader_, config);
   if (!status.ok()) {
-    LOG(QFATAL) << "Could not load enclave " << enclave_path_ << ": " << status;
+    LOG(ERROR) << "Could not load enclave " << enclave_path_ << ": " << status;
+    return status.ToOtherStatus<grpc::Status>();
   }
   LOG(INFO) << "Enclave created";
+  return grpc::Status::OK;
 }
 
-oak::InitializeOutput OakManager::GetEnclaveOutput(const std::string& node_id) {
+grpc::Status OakManager::GetEnclaveOutput(const std::string& node_id, oak::InitializeOutput* out) {
```

Comment:
Should we consider something like `StatusOr`? Not sure how well that works for gRPC status codes though.

Reply:
Yeah, I thought about that as it would be more Rust-`Result`-like, but I couldn't find a `grpc::StatusOr` or an `absl::StatusOr`.  However, looking again I've found an `asylo::StatusOr` we can use.